swayfmt/src/comments.rs

```diff
@@ -0,0 +1,66 @@
+use std::fmt::Write;
+use sway_ast::token::Comment;
+use sway_types::Spanned;
+
+use crate::{formatter::FormattedCode, Formatter, FormatterError};
+
+// Given a start, an end and a CommentMap, return references to all comments
+// contained within a given start and end of a span, in an exclusive range.
+pub fn get_comments_between(start: usize, end: usize, formatter: &mut Formatter) -> Vec<&Comment> {
```

Comment:
If I understood the usage [here](https://github.com/FuelLabs/sway/blob/d6d2fa945bd74878cdbe5b5b6720c541e34b52d4/swayfmt/src/utils/language/expr/conditional.rs#L28) correctly, I wonder if it would make sense to accept a `std::Range` and also provide a way to convert `Span` to `Range` so that it would be easy to use the function while searching in an item.

Reply:
We now accept a `std::ops::Range` in `comments_between()`.